--- ipvsadm-1.21-11/Makefile.orig	Sat Jan 22 04:07:42 2005
+++ ipvsadm-1.21-11/Makefile	Sat Jun  4 16:26:42 2005
@@ -34,9 +34,9 @@
 
 CC	= gcc
 ifeq "${ARCH}" "sparc64"
-    CFLAGS = -Wall -Wunused -Wstrict-prototypes -g -O2 -m64 -pipe -mcpu=ultrasparc -mcmodel=medlow
+    CFLAGS = $(OPT) -Wall -Wunused -Wstrict-prototypes -m64 -pipe -mcpu=ultrasparc -mcmodel=medlow
 else
-    CFLAGS = -Wall -Wunused -Wstrict-prototypes -g -O2
+    CFLAGS = $(OPT) -Wall -Wunused -Wstrict-prototypes
 endif
 SBIN    = $(BUILD_ROOT)/sbin
 MANDIR	= usr/man
@@ -45,7 +45,7 @@
 MKDIR   = mkdir
 INSTALL = install
 KERNELSOURCE = /usr/src/linux
-INCLUDE = -I$(KERNELSOURCE)/include -I.. -I.
+INCLUDE = -I.. -I.
 STATIC_LIBS = libipvs/libipvs.a
 
 
@@ -59,18 +59,12 @@
 		echo "rpm";			\
 	fi )
 
-IP_VS_H_DEFINE = $(shell if [ ! -f ../ip_vs.h ]; then \
+IP_VS_H_DEFINE = $(shell if [ ! -f ip_vs.h ]; then \
 	echo "-DHAVE_NET_IP_VS_H"; fi;)
 
 ifeq (,$(FORCE_GETOPT))
 LIB_SEARCH = /lib /usr/lib /usr/local/lib
-POPT_LIB = $(shell for i in $(LIB_SEARCH); do \
-  if [ -f $$i/libpopt.a ]; then \
-    if nm $$i/libpopt.a | fgrep -q poptGetContext; then \
-	echo "-L$$i -lpopt"; \
-    fi; \
-  fi; \
-done)
+POPT_LIB = -lpopt
 endif
 
 ifneq (,$(POPT_LIB))
@@ -91,7 +85,7 @@
 
 install:        ipvsadm
 		if [ ! -d $(SBIN) ]; then $(MKDIR) -p $(SBIN); fi
-		$(INSTALL) -m 0755 -s ipvsadm $(SBIN)
+		$(INSTALL) -m 0755 ipvsadm $(SBIN)
 		$(INSTALL) -m 0755 ipvsadm-save $(SBIN)
 		$(INSTALL) -m 0755 ipvsadm-restore $(SBIN)
 		[ -d $(MAN) ] || $(MKDIR) -p $(MAN)
