--- ipvsadm-1.26/Makefile.orig	2004-10-27 17:53:07.000000000 +0200
+++ ipvsadm-1.26/Makefile	2005-06-06 21:23:52.864458968 +0200
@@ -33,7 +33,7 @@
 ARCH		= $(shell uname -m)
 
 CC		= gcc
-INCLUDE		=
+INCLUDE		= 
 SBIN		= $(BUILD_ROOT)/sbin
 MANDIR		= usr/man
 MAN		= $(BUILD_ROOT)/$(MANDIR)/man8
@@ -43,9 +43,9 @@
 STATIC_LIBS	= libipvs/libipvs.a
 
 ifeq "${ARCH}" "sparc64"
-    CFLAGS = -Wall -Wunused -Wstrict-prototypes -g -m64 -pipe -mcpu=ultrasparc -mcmodel=medlow
+    CFLAGS = $(OPT) -Wall -Wunused -Wstrict-prototypes -m64 -pipe -mcpu=ultrasparc -mcmodel=medlow
 else
-    CFLAGS = -Wall -Wunused -Wstrict-prototypes -g
+    CFLAGS = $(OPT) -Wall -Wunused -Wstrict-prototypes
 endif
 
 
@@ -60,15 +60,7 @@
 	fi )
 
 ifeq (,$(FORCE_GETOPT))
-LIB_SEARCH = /lib64 /usr/lib64 /usr/local/lib64 /lib /usr/lib /usr/local/lib
-POPT_LIB = $(shell for i in $(LIB_SEARCH); do \
-  if [ -f $$i/libpopt.a ]; then \
-    if nm $$i/libpopt.a | fgrep -q poptGetContext; then \
-	echo "-lpopt"; \
-	break; \
-    fi; \
-  fi; \
-done)
+POPT_LIB = -lpopt
 endif
 
 ifneq (,$(POPT_LIB))
@@ -78,7 +71,5 @@
 LIBS		= $(POPT_LIB)
 DEFINES		= -DVERSION=\"$(VERSION)\" -DSCHEDULERS=\"$(SCHEDULERS)\" \
		  -DPE_LIST=\"$(PE_LIST)\" $(POPT_DEFINE)
-DEFINES		+= $(shell if [ ! -f ../ip_vs.h ]; then	\
-		     echo "-DHAVE_NET_IP_VS_H"; fi;)
 
 
--- ipvsadm-1.24/libipvs/Makefile.orig	2002-12-19 14:53:07.000000000 +0100
+++ ipvsadm-1.24/libipvs/Makefile	2005-06-06 21:25:13.995125232 +0200
@@ -1,14 +1,10 @@
 # Makefile for libipvs
 
 CC		= gcc
-CFLAGS		= -Wall -Wunused -Wstrict-prototypes -g -fPIC
+CFLAGS		= $(OPT) -Wall -Wunused -Wstrict-prototypes -fPIC
 ifneq (0,$(HAVE_NL))
 CFLAGS		+= -DLIBIPVS_USE_NL
 endif
 
-INCLUDE		+= $(shell if [ -f ../../ip_vs.h ]; then	\
-		     echo "-I../../."; fi;)
-DEFINES		= $(shell if [ ! -f ../../ip_vs.h ]; then	\
-		    echo "-DHAVE_NET_IP_VS_H"; fi;)
 
 .PHONY		= all clean install dist distclean rpm rpms
