diff -urNp -x '*.orig' ipvsadm-1.31.org/Makefile ipvsadm-1.31/Makefile
--- ipvsadm-1.31.org/Makefile	2019-12-24 14:20:30.000000000 +0100
+++ ipvsadm-1.31/Makefile	2024-04-25 23:17:44.241816304 +0200
@@ -36,7 +36,7 @@ RPMSOURCEDIR	= $(shell rpm --eval '%_sou
 RPMSPECDIR	= $(shell rpm --eval '%_specdir')
 
 CC		= gcc
-INCLUDE		=
+INCLUDE		= 
 SBIN		= $(BUILD_ROOT)/sbin
 MANDIR		= usr/man
 MAN		= $(BUILD_ROOT)/$(MANDIR)/man8
@@ -46,9 +46,9 @@ INSTALL		= install
 STATIC_LIBS	= libipvs/libipvs.a
 
 ifeq "${ARCH}" "sparc64"
-    CFLAGS = -Wall -Wunused -Wstrict-prototypes -g -m64 -pipe -mcpu=ultrasparc -mcmodel=medlow
+    CFLAGS = $(OPT) -Wall -Wunused -Wstrict-prototypes -m64 -pipe -mcpu=ultrasparc -mcmodel=medlow
 else
-    CFLAGS = -Wall -Wunused -Wstrict-prototypes -g
+    CFLAGS = $(OPT) -Wall -Wunused -Wstrict-prototypes
 endif
 
 
@@ -75,8 +75,6 @@ LIBS		+= $(shell \
 endif
 DEFINES		= -DVERSION=\"$(VERSION)\" -DSCHEDULERS=\"$(SCHEDULERS)\" \
 		  -DPE_LIST=\"$(PE_LIST)\" $(POPT_DEFINE)
-DEFINES		+= $(shell if [ ! -f ../ip_vs.h ]; then	\
-		     echo "-DHAVE_NET_IP_VS_H"; fi;)
 
 
 .PHONY	= all clean install dist distclean rpm rpms
diff -urNp -x '*.orig' ipvsadm-1.31.org/libipvs/Makefile ipvsadm-1.31/libipvs/Makefile
--- ipvsadm-1.31.org/libipvs/Makefile	2019-12-24 14:20:30.000000000 +0100
+++ ipvsadm-1.31/libipvs/Makefile	2024-04-25 23:17:44.241816304 +0200
@@ -1,7 +1,7 @@
 # Makefile for libipvs
 
 CC		= gcc
-CFLAGS		= -Wall -Wunused -Wstrict-prototypes -g -fPIC
+CFLAGS		= $(OPT) -Wall -Wunused -Wstrict-prototypes -fPIC
 ifneq (0,$(HAVE_NL))
 CFLAGS		+= -DLIBIPVS_USE_NL
 CFLAGS		+= $(shell \
@@ -13,10 +13,6 @@ CFLAGS		+= $(shell \
 		fi)
 endif
 
-INCLUDE		+= $(shell if [ -f ../../ip_vs.h ]; then	\
-		     echo "-I../../."; fi;)
-DEFINES		= $(shell if [ ! -f ../../ip_vs.h ]; then	\
-		    echo "-DHAVE_NET_IP_VS_H"; fi;)
 DEFINES		+= $(shell if which pkg-config > /dev/null 2>&1; then \
 			 if   pkg-config --exists libnl-3.0; then :; \
 			 elif pkg-config --exists libnl-2.0; then :; \
